src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
Please remove class that is no longer required from commit

Reply:
Removed.